test/nuts/plugin-install.nut.ts

```diff
@@ -18,6 +18,12 @@ let session: TestSession;
 describe('plugins:install commands', () => {
   before(async () => {
     session = await TestSession.create();
+    await fs.writeJson(
```

Comment:
@mshanemc I thought you said this file needed to be named `.sfdx/acknowledgedUsageCollection.json`?